llvm/docs/NVPTXUsage.rst

```diff
@@ -685,14 +685,22 @@ Syntax:
 Overview:
 """""""""
 
-The '``@llvm.nvvm.discard.*``'  invalidates the data at the address range [a .. a + (size - 1)]
-abhilash1910 marked this conversation as resolved.
-in the cache level specified by the .level qualifier without writing back the data
-in the cache to the memory. The operand size is an integer constant that specifies the amount of data,
-in bytes, in the cache level specified by the .level qualifier to be discarded. The only supported value
-for the size operand is 128. If no state space is specified then Generic Addressing is used.
-If the specified address does not fall within the address window of .global state space then
-the behavior is undefined.
+The '``@llvm.nvvm.discard.*``' semantically behaves like a weak write of an *unstable indeterminate value*: 
```

Comment:
@abhilash1910 where does LLVM define what an *unstable indeterminate value* is? 
You'll need to adapt this to the LLVM abstract machine, since the semantics described here seem incompatible with it.